WalletWasabi/Synchronization/FilterMessage.cs

```diff
@@ -0,0 +1,33 @@
+using System.IO;
+using WalletWasabi.Backend.Models;
+using WalletWasabi.Extensions;
+
+namespace WalletWasabi.Synchronization;
+
+public enum RequestMessage
+{
+	BestKnowBlockHash
```

Comment:
```suggestion
	BestKnownBlockHash
```

Reply:
Done in [ba429eb](https://github.com/zkSNACKs/WalletWasabi/pull/12400/commits/ba429eba531e99fdfc8ccffef9b8ca016aca0d4a)